runtime/server/x86/decoder/ctc_prefix_beam_search.cc

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Mobvoi Inc. All Rights Reserved.
+// Author: binbinzhang@mobvoi.com (Binbin Zhang)
+
+#include <glog/logging.h>
```

Comment:
1. Sort headers. Standard library headers should be put before all other third party headers
2. Use double quotes `""`, not `<>`, for non-standard headers
3. `#include "decoder/ctc_prefix_beam_search.h"` should be the first one,  before all other headers, including headers from STL.